MyFiles/Class.cs

```diff
@@ -0,0 +1,63 @@
+﻿//Class.cs
+
+using System.ComponentModel.Design;
+
+namespace VendingMachineManager
+{
+    public class VendingMachine
```

Comment:
Unless your learning material tells you to do otherwise (which applies to any comment I leave on this review) rename your file `Class.cs` → `VendingMachineManager.cs`.

Best practices for .NET are to name your files after the class that the file defines, and nest those files in directories whose names are the name of the enclosing namespace. So instead of `MyFiles\Class.cs` it would be `VendingMachineManager\VendingMachine.cs`.